docs/components/table-view.md

```diff
@@ -352,6 +366,30 @@ const getRowLink = (row: Record<string, any>): RowLink => ({
 </script>
 ```
 
+### rowKey
+
+Certain features of KTableView, such as bulk actions, require a unique identifier for each row of table data. The `rowKey` prop provides a way to define which property of the row object should serve as this unique identifier or to generate a custom unique value for each row. This prop accepts either a string or a function: `string | ((row: Record<string, any>) => string)`.
```

Comment:
docs: can we explicitly call out the features of the table that will utilize the `rowKey` prop instead of generalizing here?